src/Html.php

```diff
@@ -938,17 +937,23 @@ public static function progressBar($id, array $options = [])
      * Create a Dynamic Progress Bar
      *
      * @param string $msg  initial message (under the bar)
+     * @param array  $options See {@link Html::progressBar()} for available options (excluding message)
      *
-     * @return void
+     * @return string|void
      **/
-    public static function createProgressBar($msg = "&nbsp;")
+    public static function createProgressBar($msg = "&nbsp;", array $options = [])
```

Comment:
The `$msg` parameter could have the `null` value by default, it would simplify the following code.
```suggestion
    public static function createProgressBar($msg = null, array $options = [])
```